Indexing/ClauseCodeTree.cpp

```diff
@@ -258,23 +256,21 @@ void ClauseCodeTree::remove(Clause* cl)
   incTimeStamp();
 
   CodeOp* op=getEntryPoint();
-  firstsInBlocks.push(op);
+  firstsInBlocks->push(op);
   unsigned depth=0;
   for(;;) {
-    RemovingLiteralMatcher* rlm;
-    Recycler::get(rlm);
-    rlm->init(op, lInfos.array(), lInfos.size(), this, &firstsInBlocks);
-    rlms.push(rlm);
+    rlms->push(Recycled<RemovingLiteralMatcher, NoReset>());
+    RemovingLiteralMatcher* rlm = &*rlms->top();
+    rlm->init(op, lInfos.array(), lInfos.size(), this, &*firstsInBlocks);
 
   iteration_restart:
     if(!rlm->next()) {
       if(depth==0) {
 	ASSERTION_VIOLATION;
 	INVALID_OPERATION("clause to be removed was not found");
       }
-      Recycler::release(rlms.pop());
```

Comment:
Why don't we need the `pop` anymore?

Reply:
hmmmm interesting .. this is indeed strange, and probably a bug. I'm very surprised that this didn't show up in our test runs .. is this code actually being executed in them? 🤔

Reply:
Some of the code tree stuff is known to be dead code, so it could just not be. Jakob was thinking about reviving it, but I guess with his recent direction probably not.

Reply:
OK, fixing...